### DISCUSSION TITLE
hgv-no routing restriction

### DISCUSSION THREAD

**Discussion by rkleineisel** [osmandapp/OsmAnd#15212] [Maps Related]
When I use a routing profile with a vehicle weight of over 3.5 t osmand routes me over roads with the tag "hgv=no". In Germany roads with the hgv=no sign are forbidden for trucks over 3.5 t legal weight limit.

Other OSM street access restrictions like maxweight and maxheight are used correctly.

Is this a bug or am I using this feature incorrectly?

I tested on osmand 4.1.11 on android.

**Comment by dmpr0** [comment]
It was fixed in 4.2 release https://github.com/osmandapp/OsmAnd/issues/13126
